Resources/public/js/jquery.mjs.nestedSortable.js

```diff
@@ -0,0 +1,910 @@
+/*
```

Comment:
https://github.com/GrossumUA/StaticPageBundle/issues/7

I think, it will be better as improvement